src/pages/Address/Address.js

```diff
@@ -15,9 +16,21 @@ const POST_CODE_STYLE = {
 class Address extends React.Component {
   state = {
     isPost: false,
+    isDelivery: '',
   };
+
   handleComplete = data => {
     let zonecode = data.zonecode;
+    this.setState({ isPost: !this.state.isPost });
+
+    fetch(`${API.ADDRESS}`, {
+      method: 'POST',
+      body: JSON.stringify({
+        zipCode: zonecode,
+      }),
+    })
+      .then(res => res.json())
+      .then(result => this.setState({ isDelivery: result.message }));
```

Comment:
- 따로 setState 를 처리하는 것 보다, then 구문안 setStated에서 한번에 처리하는게 더 나은 것 같습니다!
```suggestion
    fetch(`${API.ADDRESS}`, {
      method: 'POST',
      body: JSON.stringify({
        zipCode: zonecode,
      }),
    })
      .then(res => res.json())
      .then(result => this.setState({ 
         isDelivery: result.message,
         isPost: !this.state.isPost 
       }));
```

Reply:
수정 했습니다!